### DISCUSSION TITLE
choose layers to display and identify

### DISCUSSION THREAD

**Discussion by AliceHuangn** [opengeos/leafmap#236] [Q&A] [answered]
I have multiple shapefiles and I upload them to the leafmap. Now, I want to choose the specific layer to display and identify it. How to do that in the leafmap?

**Comment by giswqs** [comment, marked as answer]
You can click the toolbar button in the upper-right corner to toggle layers on/off.

**Reply by AliceHuangn** [reply]
It works. Thank you very much. Besides, how can I modify the transparency of my shapefile, for example, I want the lines and the inside of my polygons to be transparent in the leafmap. How can I do that in the leafmap?

**Reply by giswqs** [reply]
See the example and https://github.com/giswqs/leafmap/issues/28

```python
import leafmap
m = leafmap.Map(center=[0, 0], zoom=2)
url = "https://raw.githubusercontent.com/giswqs/leafmap/master/examples/data/countries.geojson"
style={
     'color': 'red', 'fillOpacity': 0, 'weight': 1
}

m.add_geojson(
    url, layer_name="Countries", style=style
)
m
```
